test.py

```diff
@@ -60,6 +60,41 @@ def test_get_experiment_info(self):
                 test_info["ExperimentClass"].model_dump()
             )
 
+    def test_get_experiment_queue(self):
+        test_queues = tuple({
+            '1': {'pipeline': 'main',
+                'expid': {
+                    'log_level': 30,
+                    'class_name': None,
+                    'arguments': {'user': 'QuIQCL', 'time': 1.0, 'save': False, 'color': 'r'},
+                    'file': 'DIRECTORY-PATH'
+                },
+                'priority': 1,
+                'due_date': None,
+                'flush': False,
+                'status': status,
+                'repo_msg': None
+            }
+        } for status in ['pending', 'preparing', 'running', 'run_done', 'analyzing', 'deleting'])
+        with TestClient(main.app) as client:
+            for queue in test_queues:
+                self.mocked_client.get_status.return_value = queue
+                response = client.get("/experiment/queue/")
+                self.assertEqual(response.status_code, 200)
+                self.assertEqual(response.json(), queue)
+
+    def test_delete_experiment(self):
+        test_rid = 2**31 - 1
```

Comment:
Is there any reason to set `test_rid` to a big number?

Reply:
Anything is okay. Any suggestions?

Reply:
There is no problem! But I think a small interger looks better e.g., 0 or 1.